src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegService.kt

```diff
@@ -153,6 +153,10 @@ class StegService(
         return gammelFrist
     }
 
+    fun henleggBehandlingSteg(behandling: Behandling) {
```

Comment:
Det denne egentlig gjør er å sette alle steg til avbrutt.
Kan vi få til et bedre navn?

Reply:
@UyQuangNguyen Avbrutt bruker vi kun for henleggelse, så det gir mening til å kalle det som henleggBehandlingSteg, synes jeg.  Har du et bedre forslag?

Reply:
Noe så enkelt som `SettAlleStegTilAvbrutt`, altså kalle den hva den gjør? 😛